### PR TITLE
feat(active-release): Send 1 hour deploy summary notification

### PR DESCRIPTION
**Pull request by scttcper**
The notification is sent 1 hour after a deploy is finished via a task that runs every 5 minutes

email
Screen Shot 2022-07-06 at 6 00 13 PM

slack
Screen Shot 2022-07-06 at 6 00 27 PM

blocked by #36499

### CODE REVIEW

**Comment by scttcper** [comment]
Waiting for #36499 to deploy before merging this one

**Review by barkbarkimashark — APPROVED** [review]

**Comment by barkbarkimashark** [comment]
@scttcper hold off on merging this. I need to notify OPS to add the index manually.

File: src/sentry/templates/sentry/emails/activity/release_summary.html
Comment (leeandher) [inline src/sentry/templates/sentry/emails/activity/release_summary.html:17]: This text appears above a list of projects which I think can be a bit confusing to a reader, since the button to actually take them to the issues is not near this. Probably just me being nitpicky though, it's not a blocker.

File: tests/sentry/mail/activity/test_release_summary.py
Comment (leeandher) [inline tests/sentry/mail/activity/test_release_summary.py:70]: Great catch for testing!

**Review by leeandher — APPROVED** [review]
Code looks good! Left a few questions/comments though, but nothing to block a merge 🚀

**Comment by barkbarkimashark** [comment]
@scttcper index has been added. Feel free to merge and deploy when you get a chance.

**Pull request merged**
